js/components/discussions/threads-form.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+<form role="form" class="clearfix animated" v-on:submit.prevent="submit">
+    <div class="form-group">
+        <label for="title-new-discussion">{{ _('Title') }}</label>
+        <input type="text" id="title-new-discussion" v-model="title" class="form-control" required />
+        <label for="comment-new-discussion">{{ _('Comment') }}</label>
+        <textarea id="comment-new-discussion" v-model="comment" class="form-control" rows="3" required></textarea>
+    </div>
+    <button type="submit" class="btn btn-primary btn-block pull-right submit-new-discussion">
```

Comment:
An ergonomic improvement would be to display the button only when:
- both comment and title are set
- there is no pending submission (to avoid double submit)